tests/test_config.py

```diff
@@ -62,6 +62,42 @@ def test_valid_toml_two(
     assert not output.err
 
 
+def test_toml_style(
+    capsys: pytest.CaptureFixture[str], monkeypatch: pytest.MonkeyPatch
+) -> None:
+    """Test a correct toml with write = False."""
+    monkeypatch.chdir(CONFIG_DATA / "valid_toml_two")
+    pydocstringformatter.run_docstring_formatter(["test_package"])
```

Comment:
See comments below, but I think it's better if you use `_Run` here as you can then do:

`run = _Run(["test_package"])`
`assert run.config == ...`

The tests should also be moved to the `TestStyleOption` which gives other examples of how to do this 😄